### ISSUE TITLE
Can't connect to candles socket.

### ISSUE THREAD

**Issue by Safojas** [burakoner/OKEx.Net#100]
OKX was change parameters to 
![изображение](https://github.com/burakoner/OKEx.Net/assets/5460847/9d66116a-8aa4-44ce-9b1a-ae07477d884d)

**Comment by burakoner** [comment]
Yes I know. This (OKEx.Net) repository is obsolete. Please migrate to OKX.Api https://github.com/burakoner/OKX.Api
